pkg/ddc/juicefs/operations/base.go

```diff
@@ -184,8 +189,16 @@ func (j JuiceFileUtils) Mkdir(juiceSubPath string) (err error) {
 	return
 }
 
-// DeleteDirs delete dir in pod
-func (j JuiceFileUtils) DeleteDirs(dirs []string) (err error) {
+// DeleteCacheDirs delete cache dir in pod
+func (j JuiceFileUtils) DeleteCacheDirs(dirs []string) (err error) {
+	for _, dir := range dirs {
+		// cache dir check
+		match := ValidCacheDir(dir)
+		if !match {
+			err = fmt.Errorf("dir %s is not valid", dir)
```

Comment:
How about using the following info？

```golang
err = fmt.Errorf("invalid cache directory %s, skip cleaning up", dir)
```

Reply:
done

Reply:
@zwwhdls The code now returns an error but I think we should ignore the error if it's not a valid cache dir.